src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -197,39 +197,31 @@ function presentTransaction (transaction) {
 const HoloFuelDnaInterface = {
   user: {
     get: async () => {
-      const myProfile = await createZomeCall('profile/get_my_profile')()
-      if (myProfile.Err) throw new Error(`There was an error locating the current holofuel agent profile. ERROR: ${myProfile.Err}.`)
+      const { agent_address: id, avatar_url: avatarUrl, nickname, Err } = await createZomeCall('profile/get_my_profile')()
+      if (Err) throw new Error(`There was an locating your holofuel agent ID. ERROR: ${Err}.`)
```

Comment:
you accidentally a word

Reply:
I love the way you really emphasized by example! :joy: